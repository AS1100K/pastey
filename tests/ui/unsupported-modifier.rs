paste::item! {
    fn [<a:upper>]() {}
}

fn main() {}
