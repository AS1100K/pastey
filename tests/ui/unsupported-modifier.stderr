error: unsupported modifier
 --> $DIR/tests/ui/unsupported-modifier.rs:4:11
  |
4 |     fn [<a:pillow>]() {}
  |           ^^^^^^^
